Activities/Cryptography/UiPath.Cryptography.Activities/KeyedHashFile.cs

```diff
@@ -84,6 +78,16 @@ protected override void CacheMetadata(CodeActivityMetadata metadata)
                 var error = new ValidationError(Resources.FipsComplianceWarning, true, nameof(Algorithm));
                 metadata.AddValidationError(error);
             }
+            if (Algorithm.ToString().StartsWith("HMAC") && Key == null && KeySecureString == null)
```

Comment:
`HMAC` should be a const somewhere, it's not a good idea to have it as a magic string in code, especially since it's reused.